estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,62 @@
+from odoo import models, fields, api, exceptions, _
+
+
+class EstatePropertyOffer(models.Model):
+    _name = 'estate.property.offer'
+    _description = 'Offer to buy the property'
+    _order = 'price desc'
+
+    price = fields.Float()
+    status = fields.Selection(
+        string='Status',
+        selection=[('accepted', 'Accepted'), ('refused', 'Refused')],
+        copy=False,
+    )
+    partner_id = fields.Many2one('res.partner', required=True)
+    property_id = fields.Many2one('estate_property', required=True)
+    validity = fields.Integer(default=7)
+    date_deadline = fields.Date(compute='_compute_offer_date_deadline', inverse='_inverse_offer_date_deadline')
+    property_type_id = fields.Many2one(related='property_id.property_type_id')
+
+    _check_offer_price = models.Constraint(
+        'CHECK(price > 0)',
+        'Offer Price cannot be 0 or below 0'
```

Comment:
```suggestion
        'The offer price must be greater than 0'
```
I think it is littlebit simpler. What do you think?